identity-actor/src/tests/actor.rs

```diff
@@ -0,0 +1,343 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::sync::atomic::AtomicBool;
+use std::sync::Arc;
+use std::task::Poll;
+
+use futures::pin_mut;
+use libp2p::request_response::OutboundFailure;
+use libp2p::Multiaddr;
+
+use crate::actor::Actor;
+use crate::actor::ActorBuilder;
+use crate::actor::Endpoint;
+use crate::actor::Error;
+use crate::actor::ErrorLocation;
+use crate::actor::RequestContext;
+use crate::actor::Result as ActorResult;
+use crate::actor::SyncActorRequest;
+use crate::remote_account::IdentityGet;
+use crate::remote_account::IdentityList;
+use crate::tests::try_init_logger;
+
+use super::default_listening_actor;
+use super::default_sending_actor;
```

Comment:
mega nit: It is more idiomatic not to import functions.

Reply:
What would you say is the alternative, use the fully qualified path wherever they're needed?

Generally I would agree, but these functions specifically are used very often (in every test function), and so it's just more convenient to refer to them explicitly. I suppose this is a similar argument as to why the `Result::Ok` and `Result::Err` enum variants are in the default Rust prelude, rather than requiring the `Result` prefix.

Reply:
> What would you say is the alternative, use the fully qualified path wherever they're needed?

I would refer to them by `super` (i.e. `super::default_listening_actor`), but if they are really used in every single test I understand that it is less ideal to always type super so I can agree to use a full import in this instance (one typically does `use super::*` in tests anyway).